pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "mm_image_processing"
+version = "0.1.0"
+packages = [] # add required pacakges later?
+description = "IP service for MM team in E4E"
+authors = ["E4E - Mangrove Monitoring SWE Team"]
+
+# poetry entrypoint for running our Flask server
+[tool.poetry.scripts]
+image-processing-server = "mm_image_processing.server:main"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Flask = "2.3.1"
```

Comment:
This is extremely limiting, are you sure? Did you mean `^2.3`?

Reply:
Updated